cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -502,6 +502,10 @@ mkConsensusTracers trSel verb tr nodeKern fStats = do
               traceI trmet meta "submissions.rejected.count" =<<
                 STM.modifyReadTVarIO tSubmissionsRejected (+ ptxcRejected processed)
```

Comment:
I know it's unrelated to the patch here, but using `STM` for counters will have performance penalty. Using something like `ekg`'s `Gauge` for that should be better.